### PR TITLE
added  intelligence  to the sentence

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #8463 , so i added the word **Intelligence**  and also i arranged the sentences in order as per GPO admx file.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me. Nice job.

File: windows/security/threat-protection/microsoft-defender-antivirus/manage-protection-update-schedule-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -61,10 +61,10 @@ You can also randomize the times when each endpoint checks and downloads protect
 
 4.  Click **Policies** then **Administrative templates**.
 
-5.  Expand the tree to **Windows components > Microsoft Defender Antivirus > Signature Updates** and configure the following settings:
+5.  Expand the tree to **Windows components > Microsoft Defender Antivirus > Signature Intelligence Updates** and configure the following settings:
 
-    1.  Double-click the **Specify the interval to check for security intelligence updates** setting and set the option to **Enabled**. Enter the number of hours between updates. Click **OK**.
-    2. Double-click the **Specify the day of the week to check for security intelligence updates** setting and set the option to **Enabled**. Enter the day of the week to check for updates. Click **OK**.
+    1. Double-click the **Specify the day of the week to check for security intelligence updates** setting and set the option to **Enabled**. Enter the day of the week to check for updates. Click **OK**.
+    2. Double-click the **Specify the interval to check for security intelligence updates** setting and set the option to **Enabled**. Enter the number of hours between updates. Click **OK**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/manage-protection-update-schedule-microsoft-defender-antivirus.md:67]: Sorry, just a very minor nitpick: whitespace removal
```suggestion
    2. Double-click the **Specify the interval to check for security intelligence updates** setting and set the option to **Enabled**. Enter the number of hours between updates. Click **OK**.
```

**Comment by illfated** [comment]
Thank you for the chuckle I got from reading this PR title out of context.
To me, it literally seemed to say "This sentence needs more intelligence." 😄
(Not to mention that it apparently needed more cowbell too... 😉 🤣  ;)

File: windows/security/threat-protection/microsoft-defender-antivirus/manage-protection-update-schedule-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -107,4 +107,3 @@ See the following for more information and allowed parameters:
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/manage-protection-update-schedule-microsoft-defender-antivirus.md:109]: 2 blank lines after the last text should be enough. I suggest we remove these lines as well.
```suggestion
```

**Review by illfated — APPROVED** [review]

**Pull request merged**